### PR TITLE
fleetctl: Should return fail if give wrong arg to destroy or unload

### PR DESCRIPTION
**Pull request by wuqixuan**
Should return fail if give wrong arg to destroy or unload.

### CODE REVIEW

**Comment by carmstrong** [comment]
:+1:

File: fleetctl/unload.go [hunk 1]
```diff
@@ -42,6 +42,11 @@ func runUnloadUnit(args []string) (exit int) {
 		return 1
 	}
 
+	if len(units) == 0 {
+		stderr("No correct unit is found.")
```
Comment (carmstrong) [inline fleetctl/unload.go]: Maybe "%s not found" if we have the unit name here?

Comment (wuqixuan) [inline fleetctl/unload.go]: Hi, @carmstrong, thanks for your comments. The scenario is no correct units are found. If some are found, some cannot be found, it's another scenario, we can give another patch.

Comment (scole-scea) [inline fleetctl/unload.go]: _correct_ still feels like the wrong word (i.e.: who is the judge?). I'd phrase this as: "The specified unit or units were not found."

**Comment by bcwaldon** [comment]
I don't think I buy this one. Why would you fail if none of the units are found vs some of the units aren't found? Why is one scenario worse?

**Comment by wuqixuan** [comment]
This is my fault to get the two modification to the same patch. Let's talk one by one.
1. For destroy.go  I think just adding one warning is ok. How about you?
2. For unload.go ?

**Comment by dongsupark** [comment]
This is already done.
See commit 9530eedb ("fleetctl: block while destroying units to ensure units are destroyed") as well as b445699e ("fleetctl: print error message when units were not found in registry").
So I'm closing it. Thanks.

**Pull request closed without merging**